### PR TITLE
Bumping version and updating changelog

### PR DESCRIPTION
**Pull request by DuncanFenning**
Need to bump the version before tagging so that instances of the client can pass in authentication
- [x] Looks good
- [x] OK

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,2 @@
 2015-11-07 - Initial release
+2015-12-17 - Adding authentication
```
Comment (DuncanFenning) [inline CHANGELOG.md]: I know this isnt todays date but I thought the actual date of when the functionality was added would be more appropriate

Comment (shackpank) [inline CHANGELOG.md]: Can we include the version that introduced this change here as well as the date?

**Comment by pmcnr-hx** [comment]
:+1: :shipit:

**Comment by connormeredith** [comment]
:+1:

**Pull request merged**